src/pages/EditOrderPage.js

```diff
@@ -406,29 +414,35 @@ const EditOrderPage = () => {
             )}
 
             {deliveryMethod === 'delivery' && (
-              <Box sx={editOrderClasses.infoField}>
-                <Typography variant='body1' color='#031D44'>
-                  <b>Taxa de entrega:</b>
-                </Typography>
-                <Box sx={editOrderClasses.deliveryField}>
-                  <Typography variant='body1' gutterBottom sx={{ textDecoration: isElegibleForFreeDelivery() && 'line-through', mr: 1 }}>
+              <>
+                {!isElegibleForFreeDelivery() && (
+                  <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', mb: 1, mt: 4 }}>
+                    <Typography variant='body2' color='#031D44' sx={{ mr: 1, maxWidth: '350px' }}>
+                      Entrega grátis a partir de {wasTakeAwayOrder() ? amountForFreeDelivery : order.amountForFreeDelivery}€. Valor em falta: {getMissingAmountForFreeDelivery()}€.
+                    </Typography>
+                  </Box>
+                )}
+
+                <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+                  <Typography variant='h6' color='#031D44' sx={{ fontWeight: 'bold', mr: 1 }}>
+                    Taxa de entrega:
+                  </Typography>
+                  <Typography variant='body1' color='#031D44' sx={{ textDecoration: isElegibleForFreeDelivery() && 'line-through', mr: 1 }}>
                     {wasTakeAwayOrder() ? orderDeliveryFee : order.deliveryFee}€
                   </Typography>
                   {isElegibleForFreeDelivery() && (
-                    <Typography variant='body1' gutterBottom>
+                    <Typography variant='body1' color='#031D44'>
```

Comment:
Move this color to a global file with themes unless if this is a veeery specific color. It will be easier to maintain and if you want to change the theme later

Reply:
Exactly!! The funny part of this one is that i already have a theme where the colors live and i have no idea why i was not using it.